### DISCUSSION TITLE
Any way to include matched package in multiple PRs?

### DISCUSSION THREAD

**Discussion by mikeamarkey** [renovatebot/renovate#20075] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I understand that when a single package matches multiple groups, the package is included in the PR for the last matched group.

For example, the package `eslint-plugin-storybook` would ultimately be included in the PR for the Storybook group, as it is the last group that matches.
```
{
  "groupName": "Linting",
  "matchPackagePatterns": ["eslint"],
},
{
  "groupName": "Storybook",
  "matchPackagePatterns": ["storybook"],
},
```

I was wondering if there is any way to configure Renovate so that in the above example the `eslint-plugin-storybook` package would appear in both the Linting and the Storybook PRs? 

### What you tried so far.

Searching the docs, issues, and discussions

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
No, that's not possible

**Reply by mikeamarkey** [reply]
Got it, thanks for the quick reply.
Just wondering, is it A) not possible just in the current configuration sense of things, or is it B) not possible more in the sense of "it goes against the way renovate fundamentally works?"

**Reply by rarkins** [reply]
Essentially the latter. Right now updates go into one PR and aren't duplicated. It would be a moderate-high level of redesign to support it
